### PR TITLE
feat: support import result notify

### PR DESCRIPTION
**Pull request by caoxing9**

### CODE REVIEW

File: apps/nestjs-backend/src/features/notification/notification.service.ts [hunk 1]
```diff
@@ -113,6 +114,57 @@ export class NotificationService {
...
 
+  async sendImportResultNotify(params: {
+    tableId: string;
+    baseId: string;
+    toUserId: string;
+    message: string;
+  }) {
+    const { toUserId, tableId, message, baseId } = params;
+    const notifyId = generateNotificationId();
+    const [toUser] = await Promise.all([this.userService.getUserById(toUserId)]);
```
Comment (tea-artist) [inline apps/nestjs-backend/src/features/notification/notification.service.ts]: simply use await

File: apps/nestjs-backend/src/features/import/open-api/import-open-api.service.ts [hunk 2]
```diff
@@ -86,6 +92,22 @@ export class ImportOpenApiService {
...
+              toUserId: userId,
+              message: `${table.name}, import successfully🎉`,
+            });
+          },
+          (error) => {
+            this.notificationService.sendImportResultNotify({
+              baseId,
+              tableId: table.id,
+              toUserId: userId,
+              message: `${table.name}, import failed reason: ${error}`,
```
Comment (tea-artist) [inline apps/nestjs-backend/src/features/import/open-api/import-open-api.service.ts]: error.message is enough

File: apps/nestjs-backend/src/features/import/open-api/import-open-api.service.ts [hunk 3]
```diff
@@ -86,6 +92,22 @@ export class ImportOpenApiService {
             } catch (e) {
               this.logger.error((e as Error)?.message, 'Import: Records');
```
Comment (tea-artist) [inline apps/nestjs-backend/src/features/import/open-api/import-open-api.service.ts]: this.logger.error((e as Error)?.message, (e as Error)?.stack);

this way can track origin stack

File: apps/nestjs-backend/src/features/import/open-api/import-open-api.service.ts (hunk 2 above)
Comment (caoxing9) [reply apps/nestjs-backend/src/features/import/open-api/import-open-api.service.ts]: > error.message is enough

the error is message

**Review by tea-artist — APPROVED** [review]

**Pull request merged**